### DISCUSSION TITLE
BUG(Web): Select multiple assets does not work.

### DISCUSSION THREAD

**Discussion by JW-CH** [immich-app/immich#7272] [Feature Request]
### The bug

On the search page, the feature to select multiple assets (holding shift) does not work. On the photo timeline, this feature works.

![image](https://github.com/immich-app/immich/assets/17313367/1993b0cf-220c-4b18-9a0d-0e395aa42e9a)
Image description: I clicked the first image and held down shift to select multiple, this is on the /photos route.

### The OS that Immich Server is running on

UnRaid

### Version of Immich Server

v1.95

### Version of Immich Mobile App

not needed

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
not needed
```

### Your .env content

```Shell
not needed
```

### Reproduction steps

```bash
1. Open the search
2. Search for anything
3. Select an asset and hold down shift
4. Multiselect is not working
```

**Comment by e-patrick** [comment]
Discovered the same behavior and also considered it a bug, but I can also understand why this may be a FR. However, I'd really appreciate to be able to select multiple photos from the search results and, for example, add these to an album.

**Reply by maxshcherbina** [reply]
Agreed, I thought this was a bug as well but may need to be implemented differently in the code.

Go to #9159 for further discussion on this request.

**Comment by bo0tzz** [comment]
Duplicate of #6162
